terraform/api_gateway.tf.json.template.py

```diff
@@ -345,24 +356,26 @@ def for_domain(cls, domain):
                         }
                     }
                 }
+            },
+            'aws_lambda_function_event_invoke_config': {
+                function_name: {
+                    'function_name': '${aws_lambda_function.indexer_%s.function_name}' % function_name,
+                    'maximum_retry_attempts': 0
+                }
+                for function_name in (
```

Comment:
```suggestion
                for function_name in [
```

and so on where applicable according to our conventions.